server/metadata/tenant.go

```diff
@@ -127,7 +127,18 @@ func NewTenantManager(kvStore kv.TxStore, searchStore search.Store, txMgr *trans
 }
 
 func newTenantManager(kvStore kv.TxStore, searchStore search.Store, mdNameRegistry *NameRegistry, txMgr *transaction.Manager) *TenantManager {
-	collectionsInSearch, err := searchStore.AllCollections(context.TODO())
+	var collectionsInSearch map[string]*tsApi.CollectionResponse
+	var err error
+	for i := 1; i <= 5; i++ {
```

Comment:
Is it tests leftover?

Reply:
This is intended change. mainly this will be exercised in test suite where the tigris_server becomes ready before the tigris_search

Reply:
ah, ok.
Move it to the function then?
Also, the `make run` indeed consistently failing for me due to search not up.
I was just adding sleep in the docker compose:
```
bash -c '[ ! -f /etc/foundationdb/initialized ] && fdbcli --exec "configure new single memory" && touch /etc/foundationdb/initialized; sleep 3;
        /server/service'
```
It it works for you then it's better not having a hack in the code.

Reply:
Makes sense to keep it out of code. I moved it out. Thanks! Also made it 5s as I noticed that 3rd attempt succeeded in most of my runs.